ThibHlln/hydeva
# Release v0.1.0

General:
- drop support for Python 2.7.x

API changes:
- `evaluator` now supports any array-like object, e.g. `numpy.array`, `list`

Tests:
- add `unittest` test suite to check that new commits do not change results
- add `doctest` tests to check that the API as presented in the docs is working
- add GitHub workflow 

Documentation:
- add a documentation website generated with Sphinx
